src/Commands/FrequencySetCommand.php

```diff
@@ -23,7 +23,7 @@ class FrequencySetCommand extends AutopilotCommandBase implements RequestAwareIn
      * @authorize
      * @filter-output
      *
-     * @command autopilot:frequency:set
+     * @command site:autopilot:frequency
      * @aliases ap-frequency-set
```

Comment:
It feels like the alias should change too.
```suggestion
     * @aliases ap-frequency
```

Reply:
@CdrMarks will address this in https://github.com/pantheon-systems/terminus-autopilot-plugin/pull/4